README.md

```diff
@@ -109,6 +109,8 @@ Building Xamarin.Android requires:
 * [Autotools (`autoconf`, `automake`, etc.)](#autotools)
 * [`xxd`](#xxd)
 * [The Android SDK and NDK](#ndk)
+* [`brew programs`](#brew)
```

Comment:
`brew` is *not* required, in that it is not needed for a build. Nothing within the xamarin-android build system invokes it.

`build-tools/scripts/RequiredPrograms.targets` will *suggest* a `brew install` command, but it is not *executed*, and need not be "required."